website/content/en/docs/add-ons/load-balancer/guide.md

```diff
@@ -103,6 +114,11 @@ Set up resources required for the Load Balancer controller:
             ```
         - `kubernetes.io/role/internal-elb`. Add this tag only to private subnets.
         - `kubernetes.io/role/elb`. Add this tag only to public subnets.
+
+#### 2. Configure IAM
```

Comment:
we can just rename the Configure Load Balancer controller section to `Configure and Install Load Balancer Controller` see comment above

Reply:
done